tests/unit/db/client_tests.py

```diff
@@ -374,6 +375,22 @@ def test_create_db_via_setitem(self):
             self.client.delete_database(dbname)
             self.client.disconnect()
 
+    def test_db_updates_feed_call(self):
+        """
+        Test that db_updates() method call constructs and returns a Feed object
+        """
+        try:
+            self.client.connect()
+            db_updates = self.client.db_updates(limit=100)
+            self.assertIsInstance(db_updates, Feed)
```

Comment:
I think this would also pass if `db_updates` was an `InfiniteFeed`, but I think (given my limited Python expertise)  if it were written as `assertIs(type(db_updates), Feed)`, it would ensure it's a `Feed` but not an `InfiniteFeed`.